Tests/GRPCTests/Codegen/Serialization/SerializationTests.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  File.swift
+//  
+//
+//  Created by Stefana Dranca on 26/09/2023.
+//
+
```

Comment:
This needs to be a proper copyright header. Look at the other files in the project for a template.